products/ubuntu.md

```diff
@@ -196,17 +182,16 @@ releases:
     codename: "Trusty Tahr"
     lts: true
     releaseDate: 2014-04-17
-    eoas: 2019-04-02
     eol: 2019-04-02
     eoes: 2024-04-02
+    legacySupport: 2026-04-02
```

Comment:
Why not just set eoes to `2026-04-02` and remove the `legacySupport` custom column ? From what I understood the `legacySupport` column will never be used again, so it seems strange to add a column for a single release.

Reply:
1. It is unclear if Legacy Support is included as part of Ubuntu Pro, or is merely available to Ubuntu Pro subscribers (with perhaps an additional subscription)
2. I'm guessing 16.04 will get Legacy Support as well, but that's 2 years away. 

I'll revert the custom column changes for now, but I'm unsure about setting eoes to 2026, because of the lack of clarity.